File: GoScootAndroid/app/src/main/java/com/example/goscootandroid/Models/DTOs/Requests/Request_RegistrationDTO.kt
package com.example.goscootandroid.Models.DTOs.Requests

import android.annotation.SuppressLint
import kotlinx.serialization.Serializable

@SuppressLint("UnsafeOptInUsageError")
@Serializable
data class RequestRegistrationDTO(
    val full_name: String,
    val password: String,
    val phone_number: String
)

File: GoScootIoS/GoScootIoS/Models/DTO/Requests/Request_RegistrationDTO.swift
//
//  Request_RegistrationDTO.swift
//  GoScootIoS
//
//  Created by KumikoOumae on 8/11/25.
//

struct Request_RegistrationDTO: Encodable {
    let full_name: String
    let password: String
    let phone_number: String
}
